MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/EnggDiffraction/EnggDiffractionViewQtGUI.h

```diff
@@ -257,15 +257,15 @@ private slots:
   Ui::EnggDiffractionQtTabFocus m_uiTabFocus;
   Ui::EnggDiffractionQtTabPreproc m_uiTabPreproc;
   // Ui::EnggDiffractionQtTabFitting m_uiTabFitting;
-  EnggDiffFittingViewQtWidget *m_fittingWidget;
+  EnggDiffFittingViewQtWidget *m_fittingWidget{nullptr};
   Ui::EnggDiffractionQtTabSettings m_uiTabSettings;
 
   /// converts QList to a vector
   std::vector<std::string> qListToVector(QStringList list,
                                          bool validator) const;
 
   /// instrument selected (ENGIN-X, etc.)
-  std::string m_currentInst;
+  std::string m_currentInst{"ENGINX"};
```

Comment:
I think it's got to be about having a true single source of information about what ia member will be initialized to. If someone changed the default of `m_currentInst` in the class header but not the constructor then it's not obvious at first glance which one is used.

I think this is described well by http://stackoverflow.com/a/13662599. If a constructor allows overriding the default then initialize it in the class and similiar if we are using `= default` otherwise I would stick to the constructor init.